### ISSUE TITLE
A search with _include returns the same included resource multiple times if it matches more than once.

### ISSUE THREAD

**Issue by d0roppe** [IBM/FHIR#1557]
**Describe the bug**
In you search a resource and use the _include and the include matching resource is identical for more than one matching.  it is included more than once.  We should try to only return distinct include matching resources to reduce the data returned. 
**To Reproduce**

**Expected behavior**
a included resource should be unique for the search result and not returned multiple times. 
**Additional context**
Example would be searching the Coverage resource, and using _include=Coverage:*  the same patient could be the policyHolder and subscriber, and that patient resource should be returned only once.

**Comment by prb112** [comment]
Team discussion: 
- Core Problem: Duplicate on the Single Page
- Design work and team agreement must be reached before proceeding with the broader change of duplicates being returned cross pages.

**Comment by d0roppe** [comment]
stretch goal for sprint 21

**Comment by d0roppe** [comment]
verified that _include does only return 1 copy of the included resource, version specific. in case there is another version of that resource matching the include, the other version is also returned, so still unique.  Closing issue.
